client/joinus_client/src/components/Events/EventCard.js

```diff
@@ -37,14 +47,60 @@ export default function EventCard(props) {
     reload,
   } = props;
 
-  const { userCreateEventSubmit } = userEvents();
+  console.log("Props from EventCard.js:")
+  console.log(props)
+
+  // to duck: We need an address props - Where would they come from? the db.. yea but.. what fil... the db??.. sigh, what I mean is, what is the parent of where these props are coming in? why do I need to know? They're coming in from useAppData as a hook -joba
```

Comment:
what do you mean?

Reply:
This is my own ramblings -ignore this haha